### PR TITLE
feat: Add ubuntu-jammy example image based on existing ubuntu-focal

### LINKED ISSUE #2101: Ubuntu Jammy (22.04 LTS) Image
Would you be willing to update the Ubuntu example image to jammy (22.04 LTS)?

### PR DESCRIPTION
**Pull request by dylanmtaylor**
Closes #2101

### CODE REVIEW

File: images/ubuntu-jammy/github_agent.ubuntu.pkr.hcl [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+  provisioner "shell" {
+    environment_vars = [
+      "RUNNER_TARBALL_URL=https://github.com/actions/runner/releases/download/v${var.runner_version}/actions-runner-linux-x64-${var.runner_version}.tar.gz"
+    ]
+    inline = [
+      "sudo chmod +x /tmp/install-runner.sh",
+      "echo ubuntu | tee -a /tmp/install-user.txt",
+      "sudo RUNNER_ARCHITECTURE=x64 RUNNER_TARBALL_URL=$RUNNER_TARBALL_URL /tmp/install-runner.sh",
+      "echo ImageOS=ubuntu20 | tee -a /opt/actions-runner/.env"
```
Comment (npalm) [inline images/ubuntu-jammy/github_agent.ubuntu.pkr.hcl:167]:
```suggestion
      "echo ImageOS=ubuntu22 | tee -a /opt/actions-runner/.env"
```

File: images/ubuntu-jammy/github_agent.ubuntu.pkr.hcl [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+      version = ">= 0.0.2"
+      source  = "github.com/hashicorp/amazon"
+    }
+  }
+}
+
+variable "runner_version" {
+  description = "The version (no v prefix) of the runner software to install https://github.com/actions/runner/releases"
+  type        = string
+  default     = "2.286.1"
```
Comment (npalm) [inline images/ubuntu-jammy/github_agent.ubuntu.pkr.hcl:13]:
```suggestion
  default     = "2.292.0"
```

**Review by npalm — COMMENTED** [review]
LGTM, just a few small remarks

**Comment by dylanmtaylor** [comment]
> LGTM, just a few small remarks

Done 😃

**Comment by npalm** [comment]
AMI build is failing due to installing required depedencies for the runner.

```
==> githubactions-runner.amazon-ebs.githubrunner: E: Unable to locate package liblttng-ust0
    githubactions-runner.amazon-ebs.githubrunner: 'apt-get' failed with exit code '0'
```

Seems there is a PR open on the runners, see https://github.com/actions/runner/pull/1585

File: images/ubuntu-jammy/github_agent.ubuntu.pkr.hcl [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+      version = ">= 0.0.2"
+      source  = "github.com/hashicorp/amazon"
+    }
+  }
+}
+
+variable "runner_version" {
+  description = "The version (no v prefix) of the runner software to install https://github.com/actions/runner/releases"
+  type        = string
+  default     = "2.293.0"
```
[the remark this answers is not in the document]
Comment (dylanmtaylor) [reply images/ubuntu-jammy/github_agent.ubuntu.pkr.hcl]: Excellent, let's give it a go!

**Review by npalm — APPROVED** [review]

**Pull request merged**